lib/src/platform.dart

```diff
@@ -1,20 +1,20 @@
 library pana.platform;
 
+import 'dart:collection';
 import 'package:source_gen/generators/json_serializable.dart';
 
 import 'pubspec.dart';
 
 part 'platform.g.dart';
 
-abstract class KnownPlatforms {
-  static const String browser = 'browser';
-  static const String standalone = 'standalone';
+abstract class PlatformFlags {
+  /// Denotes a package that references Flutter in `pubspec.yaml`.
   static const String flutter = 'flutter';
-  static const String mirrors = 'mirrors';
 
-  /// Native extensions (Dart VM with C/C++ code).
-  static const String native = 'native';
+  /// Denotes a library that depends on a native extensions via `dart-ext:`
+  static const String dartExtension = 'dart-ext';
 
+  /// Denotes a library that depends on `pkg/angular` or `pkg/angular2`.
   static const String angular = 'angular';
```

Comment:
Nit: as we are storing `dart:*` in the list, maybe we shall use `package:[blah]` here too?